code/game/objects/items/devices/flashlight.dm

```diff
@@ -104,40 +104,55 @@
 			to_chat(user, "[span_warning("\The [src] isn't bright enough to see anything!")] ")
 			return
 
+		var/render_list = list()//information will be packaged in a list for clean display to the user
+
 		switch(user.zone_selected)
 			if(BODY_ZONE_PRECISE_EYES)
 				if((M.head && M.head.flags_cover & HEADCOVERSEYES) || (M.wear_mask && M.wear_mask.flags_cover & MASKCOVERSEYES) || (M.glasses && M.glasses.flags_cover & GLASSESCOVERSEYES))
 					to_chat(user, span_warning("You're going to need to remove that [(M.head && M.head.flags_cover & HEADCOVERSEYES) ? "helmet" : (M.wear_mask && M.wear_mask.flags_cover & MASKCOVERSEYES) ? "mask": "glasses"] first!"))
 					return
 
 				var/obj/item/organ/internal/eyes/E = M.get_organ_slot(ORGAN_SLOT_EYES)
+				var/obj/item/organ/internal/brain = M.get_organ_slot(ORGAN_SLOT_BRAIN)
 				if(!E)
 					to_chat(user, span_warning("[M] doesn't have any eyes!"))
 					return
 
+				M.flash_act(length = 10)//breifly apply flash affect to target
```

Comment:
Be sure to include `visual = TRUE` as it did in the past, so it doesn't actually stun people

Reply:
so THAT'S what that does. Added!